### PR TITLE
Feature/sso

### PR DESCRIPTION
**Pull request by Anthv96A**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/Sage.ico" />
```
Comment (edleeks87) [inline public/index.html:5]: 🎉

File: src/components/containers/banner/banner.component.js [hunk 2]
```diff
@@ -2,44 +2,52 @@ import React from 'react';
...
+import authProvider from '../../../azure/authProvider';
+
+const Banner = ({ auth }) => {
+  const displayText = (auth.isAuth && auth.account.name)
+    ? `back to Belts and Leaders ${auth.account.name}!`
+    : 'to Belts & Leaders, our security champion achievement league!';
+
+  const loginOrLogAchievement = (auth.isAuth)
+    ? <Button buttonType='primary'>Log Achievement</Button>
+    : <AzureAD provider={ authProvider }>{({ login }) => <Button onClick={ login } buttonType='primary'>Sign In</Button> }</AzureAD>;
```
Comment (edleeks87) [inline src/components/containers/banner/banner.component.js:20]: Good shout removing link, think we might get away with stripping the routing out all together 🤞

Comment (edleeks87) [reply src/components/containers/banner/banner.component.js:20]: Especially if we remove the option from the dropdown, might be worth raising that with Emma next week 🤔

File: src/components/containers/nav-bar/user-menu/user-menu.component.js [hunk 3]
```diff
@@ -2,37 +2,38 @@ import React from 'react';
...
-          <MenuItem href='#'>
-            View Profile
-          </MenuItem>
-          <MenuItem href='#'>
-            Logout
-          </MenuItem>
-        </MenuItem>
-      </Menu>
-    </StyledUserMenu>
+    <AzureAD provider={ authProvider }>{ctx => (
```
Comment (edleeks87) [inline src/components/containers/nav-bar/user-menu/user-menu.component.js:11]: Is it not possible to consume from the provider implemented in the root or are we going to have to implement everywhere we need it? For example we'll need it in the Profile etc, so if we can just wire up the `Consumer`s and have the single `Provider` that would be better imo. If not maybe we should write our own context provider and pass the auth down after Azure has fetched it etc

File: src/components/containers/banner/banner.component.js (hunk 2 above)
Comment (Anthv96A) [reply src/components/containers/banner/banner.component.js:20]: I hope we can get away with it like! Yeah, it's probably a good shout!

File: src/components/containers/nav-bar/user-menu/user-menu.component.js (hunk 3 above)
Comment (Anthv96A) [reply src/components/containers/nav-bar/user-menu/user-menu.component.js:11]: I tried to do this from the root last night, and I couldn't get it working unfortunately ... 

I agree that would be better, but we can take a look after the POC has passed

File: public/index.html (hunk 1 above)
Comment (Anthv96A) [reply public/index.html:5]: 👊

**Comment by DipperTheDan** [comment]
Nice work, @Anthv96A .

**Pull request merged**